generator/src/render.js

```diff
@@ -634,10 +652,9 @@ function flushIfDone(app) {
 }
 
 function flushQueue(app) {
-  const temp = pendingBackendTaskResponses;
-  pendingBackendTaskResponses = [];
+  app.ports.gotBatchSub.send(Object.fromEntries(pendingBackendTaskResponses));
+  pendingBackendTaskResponses = new Map();
```

Comment:
You can also use `.clear()`

Reply:
Yeah, I tried that originally and was running into issues, I still need to figure out exactly the ideal strategy to avoid allocating more memory than needed. Using `clear()` stepped on the toes of some other process, but I might be able to avoid that if I initialize it in a different place (right now it's initialized in the top-level of the JS file).